spectrocloud/schemas/pack.go

```diff
@@ -16,27 +16,30 @@ func PackSchema() *schema.Schema {
 					Type:     schema.TypeString,
 					Computed: true,
 					Optional: true,
+					Description: "The unique identifier of the pack. Could be looked up using the `spectrocloud_pack` data source. " +
+						"Must be specified if the pack `type` is `spectro`.",
```

Comment:
```suggestion
					Description: "The unique identifier of the pack. The value can be looked up using the [`spectrocloud_pack`](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs/data-sources/pack) data source. This value is required if the pack type is `spectro`.",
```

Added a link to the data resource docs to help the reader.